src/routing/components/remote_data/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  always, anyPass, assoc, either,
+  filter, ifElse, length, pipe, prop, reduce, toPairs, values
+} from 'ramda';
+import * as React from 'react';
+import { furl, self } from '../../../utils';
+import * as _RemoteData from '../../remote_data';
+
+import { default as _Loaded } from './loaded';
+import { default as _Loading } from './loading';
+
+export const LOADED = _Loaded;
+export const LOADING = _Loading;
+
+type Props<T> = {
+  data: { [K in keyof T]: _RemoteData.RemoteData<T[K]> };
+  loading: () => React.Component<any>;
+  loaded: (data: T) => React.Component<any>;
+  policy?: any[];
```

Comment:
There is probably a way to make policy type safe too. I see there is a typedef on the `_RemoteData.is` so probably an array of that